src/nft/components/collection/CollectionStats.tsx

```diff
@@ -234,26 +233,26 @@ const CollectionDescription = ({ description }: { description: string }) => {
 const StatsItem = ({ children, label, isMobile }: { children: ReactNode; label: string; isMobile: boolean }) => {
   return (
     <Box display="flex" flexDirection={isMobile ? 'row' : 'column'} alignItems="baseline" gap="2" height="min">
+      <span className={styles.statsValue}>{children}</span>
       <Box as="span" className={styles.statsLabel}>
         {`${label}${isMobile ? ': ' : ''}`}
       </Box>
-      <span className={styles.statsValue}>{children}</span>
     </Box>
   )
 }
 
 const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMobile?: boolean } & BoxProps) => {
-  const numOwnersStr = stats.stats ? putCommas(stats.stats.num_owners) : 0
-  const totalSupplyStr = stats.stats ? putCommas(stats.stats.total_supply) : 0
+  const numOwnersStr = stats.stats ? quantityFormatter((stats.stats.num_owners / stats.stats.total_supply) * 100) : 0
+  const totalSupplyStr = stats.stats ? quantityFormatter(stats.stats.total_supply) : 0
   const listedPercentageStr =
     stats.stats && stats.stats.total_listings > 0
-      ? ((stats.stats.total_listings / stats.stats.total_supply) * 100).toFixed(0)
+      ? Math.round((stats.stats.total_listings / stats.stats.total_supply) * 100)
       : 0
```

Comment:
We need the logic to show `<1%` for listedPercentage and for the percent of unique owners. This + the rounding logic can be moved to a util function in `numbers.ts`. 

Can the var name `numOwnersStr` be updated to `uniqueOwnersPercentage` to match new format?